### PR TITLE
Actual Wynn icons & branches for atree instead of basic circles and lines

### PR DESCRIPTION
**Pull request by fin444**

### CODE REVIEW

File: js/atree.js [hunk 1]
```diff
@@ -979,102 +986,80 @@ function render_AT(UI_elem, list_elem, tree) {
         let node_elem = document.createElement('div');
         let icon = ability.display.icon;
         if (icon === undefined) {
-            icon = "node_0";
+            icon = "node";
+        } else if (icon == "node_4") {
+            icon = "node_warrior" // temp fix
```
Comment (mass2010chromium) [inline js/atree.js:991]: is this correct?

File: js/atree.js [hunk 2]
```diff
@@ -1151,13 +1111,17 @@ function atree_render_connection(atree_connectors_map) {
 
 // toggle the state of a node.
 function atree_set_state(node_wrapper, new_state) {
+    console.log(node_wrapper.elem.children[0])
     if (new_state) {
+        console.log("on")
```
Comment (mass2010chromium) [inline js/atree.js:1116]: remove debug prints

File: js/atree.js [hunk 3]
```diff
@@ -1151,13 +1111,17 @@ function atree_render_connection(atree_connectors_map) {
...
         node_wrapper.active = true;
-        node_wrapper.elem.classList.add("atree-selected");
+        node_wrapper.elem.children[0].src = node_wrapper.elem.children[0].src.substring(0, node_wrapper.elem.children[0].src.length - 4) + "_selected.png";
+        console.log(node_wrapper.elem.children[0].src)
```
Comment (mass2010chromium) [inline js/atree.js:1119]: remove debug prints

File: js/atree.js [hunk 4]
```diff
@@ -1151,13 +1111,17 @@ function atree_render_connection(atree_connectors_map) {
...
     } 
     else {
+        console.log("off")
```
Comment (mass2010chromium) [inline js/atree.js:1122]: remove debug prints

File: js/atree.js [hunk 5]
```diff
@@ -1151,13 +1111,17 @@ function atree_render_connection(atree_connectors_map) {
 
 // toggle the state of a node.
 function atree_set_state(node_wrapper, new_state) {
+    console.log(node_wrapper.elem.children[0])
     if (new_state) {
+        console.log("on")
         node_wrapper.active = true;
-        node_wrapper.elem.classList.add("atree-selected");
+        node_wrapper.elem.children[0].src = node_wrapper.elem.children[0].src.substring(0, node_wrapper.elem.children[0].src.length - 4) + "_selected.png";
```
Comment (mass2010chromium) [inline js/atree.js:1118]: probably better to store the "node type" string -- this is really fragile

File: js/atree.js [hunk 6]
```diff
@@ -1151,13 +1111,17 @@ function atree_render_connection(atree_connectors_map) {
...
+        console.log(node_wrapper.elem.children[0].src)
     } 
     else {
+        console.log("off")
         node_wrapper.active = false;
-        node_wrapper.elem.classList.remove("atree-selected");
+        node_wrapper.elem.children[0].src = node_wrapper.elem.children[0].src.substring(0, node_wrapper.elem.children[0].src.length - 13) + ".png";
```
Comment (mass2010chromium) [inline js/atree.js:1124]: probably better to store the "node type" string -- this is really fragile

File: js/atree.js [hunk 7]
```diff
@@ -1172,6 +1136,21 @@ function atree_set_state(node_wrapper, new_state) {
...
+            connector_elem.style = "width: 100%; height: 100%;";
+            connector_elem.src = '../media/atree/connect_' + v[0].type + '.png'
+            v[0].replaceChildren(connector_elem);
+        }
+    });
+    atree_map.forEach((v) => {
+        atree_compute_highlight(v);
+    });
+}
+
```
Comment (mass2010chromium) [inline js/atree.js:1153]: i believe this is dead/unused code unless you are calling it somewhere

**Review by mass2010chromium — CHANGES_REQUESTED** [review]
sorry i do not have time to test run your code rn, but here's what i see just looking at the diff

File: js/atree.js (hunk 1 above)
Comment (fin444) [reply js/atree.js:991]: lmao whoops

File: js/atree.js (hunk 7 above)
Comment (fin444) [reply js/atree.js:1153]: It seems that this function was removed by another contributer and I missed it in merging

**Review by hppeng-wynn — APPROVED** [review]
lgtm

**Pull request merged**